src/sentry/api/serializers/models/__init__.py

```diff
@@ -1,3 +1,4 @@
+from ....notifications.api.serializers.notification_action_response import *  # noqa: F401,F403
```

Comment:
Could this be an absolute import? four `.` is a lot.

Reply:
yeeee I keep doing this 🫠 . I need a pre-commit hook or that catches > 1 dot or smthn